### PR TITLE
Add sonar-scanner

### PR DESCRIPTION
**Pull request by hazim1093**

### CODE REVIEW

**Comment by stakater-user** [comment]
@hazim1093 Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins.release.stakater.com/job/Stakater%20Docker/job/pipeline-tools/job/PR-6/1/) has Failed!

**Comment by stakater-user** [comment]
@hazim1093 Yikes! You better fix it before anyone else finds out! [Build 2](https://jenkins.release.stakater.com/job/Stakater%20Docker/job/pipeline-tools/job/PR-6/2/) has Failed!

**Comment by stakater-user** [comment]
@hazim1093 Yikes! You better fix it before anyone else finds out! [Build 3](https://jenkins.release.stakater.com/job/Stakater%20Docker/job/pipeline-tools/job/PR-6/3/) has Failed!

**Comment by stakater-user** [comment]
@hazim1093 Yikes! You better fix it before anyone else finds out! [Build 4](https://jenkins.release.stakater.com/job/Stakater%20Docker/job/pipeline-tools/job/PR-6/4/) has Failed!

**Comment by stakater-user** [comment]
@hazim1093 Image is available for testing. `docker pull docker.release.stakater.com:443/stakater/pipeline-tools:SNAPSHOT-PR-6-5`

**Comment by stakater-user** [comment]
@hazim1093 Image is available for testing. `docker pull docker.io/stakater/pipeline-tools:SNAPSHOT-PR-6-7`

**Comment by stakater-user** [comment]
@hazim1093 Image is available for testing. `docker pull docker.io/stakater/pipeline-tools:SNAPSHOT-PR-6-8`

**Comment by stakater-user** [comment]
@hazim1093 Image is available for testing. `docker pull docker.io/stakater/pipeline-tools:SNAPSHOT-PR-6-9`

**Comment by stakater-user** [comment]
@hazim1093 Yikes! You better fix it before anyone else finds out! [Build 10](https://jenkins.release.stakater.com/job/Stakater%20Docker/job/pipeline-tools/job/PR-6/10/) has Failed!

**Comment by stakater-user** [comment]
@hazim1093 Image is available for testing. `docker pull docker.io/stakater/pipeline-tools:SNAPSHOT-PR-6-12`

File: Dockerfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ ENV DOCKER_API_VERSION=1.32
 
 # Install ansible, boto, aws-cli, and some handy tools
 RUN echo "===> Installing Utilities from apk ..."  && \
-    apk -v --update --progress add sudo git bash wget openssh groff less python py-pip curl jq unzip nodejs nodejs-npm coreutils python py-pip openssl ca-certificates make sshpass openssh-client rsync gnupg gettext && \
+    apk -v --update --progress add sudo git bash wget openssh groff less python py-pip curl jq unzip nodejs nodejs-npm coreutils python py-pip openssl ca-certificates make sshpass openssh-client rsync gnupg gettext openjdk8-jre=8.191.12-r0 && \
```
Comment (rasheedamir) [inline Dockerfile:30]: isn't jdk 8 too old? why not install later version?

Comment (hazim1093) [reply Dockerfile:30]: There's no official support for jdk 11 on alpine. Although there is some port for that. 
There is now support for 12 and 13 but both are in "early access" stage

**Comment by stakater-user** [comment]
@hazim1093 Image is available for testing. `docker pull docker.io/stakater/pipeline-tools:SNAPSHOT-PR-6-14`

**Comment by stakater-user** [comment]
@hazim1093 Image is available for testing. `docker pull docker.io/stakater/pipeline-tools:SNAPSHOT-PR-6-15`

**Review by kahootali — APPROVED** [review]

**Pull request merged**